lib/include/ert/util/util.h

```diff
@@ -99,10 +99,10 @@ typedef enum {left_pad   = 0,
               center_pad = 2} string_alignement_type;
 
   //#define UTIL_CXX_MALLOC(var , num_elm) (typeof (var)) util_malloc( (num_elm) * sizeof var)
-
   void         util_bitmask_on(int *  , int );
   char       * util_get_timezone(void);
   time_t       util_make_datetime_utc(int , int , int , int , int , int );
+  bool         util_make_datetime_utc__(int sec, int min, int hour , int mday , int month , int year, time_t * t);
```

Comment:
This is not a very good function name.. How about following the `util_set_....`-syntax used by some of the other util-functions?